src/graphics/draw.rs

```diff
@@ -161,18 +161,21 @@ impl DrawParam {
         self
     }
 
+    pub(crate) fn get_dest_mut(&mut self) -> &mut mint::Point2<f32> {
+        if let Transform::Values { dest, .. } = &mut self.transform {
+            dest
+        } else {
+            panic!("Cannot calculate destination value for a DrawParam matrix")
```

Comment:
Do we perhaps want to change this to an Error?
It's not transparent that `dest` may panic on setting the destination.

Reply:
`draw_queued_text` is affected as well and already returns a `GameResult`, so no change there.
Having `dest` return one would be slightly inconvenient though (if we return one there we might as well return one on each `DrawParam` method as modifying a matrix in either of these ways doesn't make sense). So how about only actually returning the error for `draw_queued_text` and silently ignoring it in `dest`?